.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## ✍️ Proposed changes
+
+Describe the big picture of your changes here and communicate why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
```

Comment:
What I've seen other projects do is have these notes encased in HTML comment tabs. Such that they show for the submitter, but don't get persisted in the actual PR. `<!-- text -->`.

Reply:
Oh does that actually work with GitHub-flavored markdown?

Reply:
Seems like it! Check out https://github.com/emotion-js/emotion/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Hit raw to see the comments.

Reply:
Done!